profile/README.md

```diff
@@ -1,3 +1,96 @@
+* **Fork** the relevant repository.
+* **Make necessary changes** to the repository taking into account the **coding guidelines** mentioned in the individual repositories. Some general guidelines include the use of “git rebase” to organize commits and these instructions regarding commit messages:
+  * Separate subjects from body with a blank line
+  * Limit the subject line to 50 characters
+  * Capitalize the subject line
+  * Do not end the subject line with a period
+  * Use the imperative mood in the subject line
+  * Wrap the body at 72 characters
+  * Use the body to explain what and why vs. how
+* After your work is complete, **make a Pull Request** from your repository to the-deep repository. Describe what you did in as much detail as possible. Furthermore, please link to the issue in the description.
+* Our development team will go through the pull request and merge them if they are satisfactory. They can also review the PR and ask for explanations/modifications.
+
+# Repositories
+This project contains a number of repositories which work together. Here is a list of the main repositories and their brief descriptions:
+* **deeper**: The main repository for the DEEP project with configuration files and scripts for local and cloud deployments
+* **server**: Backend server with Django
+* **client**: Frontend code with React
+* **serverless**: Services for DEEP utilities that run in serverless framework
+* **deep-extensions**: Browser extension to add sources to DEEP platform
+
+# Installation Instructions
+* Clone deeper repo
+
+`git clone https://github.com/the-deep/deeper.git deep-project-root`
+* Go to deeper project root
+
+`cd deep-project-root`
+* Clone client, server and other repos
+
+`git clone https://github.com/the-deep/server.git server`
+
+`git clone https://github.com/the-deep/client.git client`
+
+`git clone --branch=feature/only-ary https://github.com/the-deep/client.git ./ary-only-client`
+
+`git clone https://github.com/the-deep/deepl-deep-integration.git deepl-service`
+
+* If you have a **Mac with M1 chip**, change the `deepl-service/Dockerfile` to specify amd64 platform in its FROM statement:
+
+`FROM --platform=linux/amd64 python:3.10-slim-buster`
+
+* If you are on a **Mac**, you have to turn off 'Airplay Reciever' from 'System Preferences > Sharing' to make port 5000 available for use.
+* Use docker-compose to start the servers (Install latest docker and docker-compose, if they are not installed)
+
+`docker-compose build`
+
+`docker-compose up`
+
+# Useful Commands
+* To migrate, go to the docker container and run migrate command:
+
+`docker-compose exec web bash`
+
+`./manage.py migrate`
+
+* To test, go to the docker container and run the test command:
+
+
+`docker-compose exec web bash`
+
+`./manage.py test`
+
+# Useful Links:
+* [Userguide](https://deephelp.zendesk.com/hc/en-us)
+* [DEEP website](https://app.thedeep.io/)
+
+
+
+
+
+
```

Comment:
Let's avoid trailing empty lines.

Reply:
Fixed in https://github.com/the-deep/.github/pull/2.